### PR TITLE
Kubernetes Image Listing

### PR DESCRIPTION
**Pull request by stefwalter**
Start of the image listing page in the kubernetes dashboard.

This pull request will only contain listing functionality. Filtering and panel expansion will happen in other pull requests.

### CODE REVIEW

**Comment by stefwalter** [comment]
Ready for review.

**Comment by stefwalter** [comment]
Fixed issue with not displaying image data, only imagestream.

File: pkg/kubernetes/images.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+                    names = names.slice(0, 5);
+                }
+
+                var format;
+                var ret = names.join(", ");
+                if (extra) {
+                    if (ret)
+                        format = cockpit.ngettext("more images", _(" + $0 more"), _(" + $0 more"), extra);
+                    else
+                        format = cockpit.ngettext(_("$0 images"), _("$0 images"), extra);
```
Comment (petervo) [inline pkg/kubernetes/images.js]: image, images?

Comment (stefwalter) [inline pkg/kubernetes/images.js]: Thanks. Fixed.

**Comment by petervo** [comment]
Maybe i'm missunderstanding the design but, something doesn't seem quite right with assigning images directly to repos. I end up with 3 listings

 * registry.access.redhat.com/openshift3/ruby-20-rhel7
 * registry.access.redhat.com/openshift3/ruby-20-rhel7:latest
 * registry.access.redhat.com/openshift3/ruby-20-rhel7:v3.0.0.0

From this json: http://pastebin.com/HbMLc9LX

**Comment by stefwalter** [comment]
@petervo Could you help me replicate your configuration. What did you 'oc create' in order to get to that point. My image streams are obviously rather tame.

**Comment by petervo** [comment]
I was following the openshift training.

https://github.com/openshift/training/blob/bb45f8f1311b20ab9bc37bdf74438da219df0d68/09-Templates-Quickstarts-Instant-Apps.md

**Comment by mvollmer** [comment]
`needswork` to prevent merging while there is a ongoing discussion.

**Comment by stefwalter** [comment]
Thanks @petervo fixed.

**Comment by stefwalter** [comment]
Added test case. This depends on #2674

**Comment by stefwalter** [comment]
Test failure is not related. Filed #2695 about it.

**Pull request closed without merging**